plugins/newsdownloader.koplugin/main.lua

```diff
@@ -97,12 +113,49 @@ function NewsDownloader:addToMainMenu(menu_items)
             },
             {
                 text = _("Settings"),
-                callback = function()
-                    UIManager:show(InfoMessage:new{
-                        text = T(_("To change feed (Atom/RSS) sources please manually edit the configuration file:\n%1\n\nIt is very simple and contains comments as well as sample configuration."),
-                                 feed_config_path)
-                    })
-                end,
+                sub_item_table = {
+                    {
+                        text = _("Change feeds configuration"),
+                        callback = function()
+                            UIManager:show(InfoMessage:new{
+                                text = T(_("To change feed (Atom/RSS) sources please manually edit the configuration file:\n%1\n\nIt is very simple and contains comments as well as sample configuration."),
+                                         feed_config_path)
+                            })
+                        end,
+                    },
+                    {
+                        text = _("Action when Wi-Fi is off"),
```

Comment:
Is there a specific reason you didn't simply copy, e.g., ReaderHighlight:genHighlightDrawerMenu()?

Reply:
Don't know what's that. I'll take a look

Reply:
It was more a generic comment about the difficulty. Unless you're creating something new you can just copy something that does what you want and manipulate it slightly. For example, https://github.com/koreader/koreader/pull/3058 is more or less a direct copy of the "sort by" setting. The difficulty was in the "start with history", not in the menu.